README.md

```diff
@@ -1,12 +1,11 @@
 #Business
 C4GNV is a member brigade of Code for America, the goal of this project is to make it easier
 to start a business in Gainesville by providing people easy access to permits and zoning information.
-The project is a light weight website written in Typescript and Angular 2, using open data sources
-as a persistent backend.
+The project is a lightweight website written in [Typescript](https://www.typescriptlang.org/) and Angular 2, using open data sources as a persistent backend.
 
-The inspiration for this service is to eventually be able to expand to be like the
```

Comment:
I think it reads strangely without "the" so if it's fine with it I'd rather keep it. The other changes seem fine though, so if you revert this change but keep the others we can merge them.